metrics/server.go

```diff
@@ -0,0 +1,13 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	PelicanServerXRootDLastCrash = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Why use a gauge vec? Time is monotonically increasing, so a counter feels more appropriate to me, but maybe this is something you've thought about more than I have.